packages/components/src/components/positions-drawer-card/positions-drawer-card.tsx

```diff
@@ -15,40 +15,37 @@ import {
     isMobile,
     isVanillaContract,
 } from '@deriv/shared';
-import { TContractInfo, TContractStore } from '@deriv/shared/src/utils/contract/contract-types';
+import { TContractStore } from '@deriv/shared/src/utils/contract/contract-types';
 import { TToastConfig } from '../types/contract.types';
 
-type TGetEndTime = Pick<
-    TContractInfo,
-    'is_expired' | 'sell_time' | 'status' | 'tick_count' | 'underlying' | 'contract_id'
-> &
-    Required<Pick<TContractInfo, 'date_expiry' | 'contract_type' | 'exit_tick_time' | 'is_path_dependent'>>;
-
 type TPositionsDrawerCardProps = {
     addToast: (toast_config: TToastConfig) => void;
     className?: string;
-    contract_info: TGetEndTime;
+    contract_info: ProposalOpenContract &
+        Portfolio1 & {
+            contract_update?: ContractUpdate | undefined;
+        };
```

Comment:
@maryia-deriv, can this `contract_update?: ContractUpdate | undefined;` be `contract_update?: ContractUpdate;` instead? 🤔